### PR TITLE
Adds new blog post.

### PR DESCRIPTION
**Pull request by claycarpenter**
Adds new blog post showing how to build a text-to-date input in Angular with ngModel parsers and formatters.

### CODE REVIEW

File: source/blog/posts/2016-03-15-build-a-text-date-input-with-ngmodel-parsers-and-formatters.html.markdown [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+title: Build a Text Date Input with ngModel parsers and formatters
+date: 2016-03-15 03:09 UTC
+tags: angular
+authors: Clay Carpenter
+---
+
+Angular's `ngModel` and its two-way binding to backing model properties is great. But what if we want the model property to be stored in a different format than what we're presenting to the user in the input? We're going to solve that problem today, with the help of `ngModel`'s parser and formatter pipelines. We'll build a new directive we can attach to any text input that will validate the user's date input, verifying that the date is formatted properly and is a real date. If the date input is valid, the directive will convert that string into a JS date. If the date input isn't valid, the directive will mark the input field as invalid.
```
Comment (nearbycoder) [inline source/blog/posts/2016-03-15-build-a-text-date-input-with-ngmodel-parsers-and-formatters.html.markdown]: > binding to backing model properties
is that supposed to be backend?

**Comment by nearbycoder** [comment]
Besides the one questionable wording comment that I had, this article was great read and also gave me a better understanding of some of the techniques we use. A+ from me. :100:

File: source/blog/posts/2016-03-15-build-a-text-date-input-with-ngmodel-parsers-and-formatters.html.markdown (hunk 1 above)
Comment (claycarpenter) [inline source/blog/posts/2016-03-15-build-a-text-date-input-with-ngmodel-parsers-and-formatters.html.markdown]: I did mean backing model properties (even though those backing model properties often do come from backends). But considering the entire article is predicated on the understanding that ngModel holds its own value separate from the actual model property it binds to, I should probably explain that a bit better. Thanks for catching that.

**Comment by claycarpenter** [comment]
@nearbycoder Updated article based on your feedback. Have a look and let me know if that tweaked introduction makes more sense.

**Comment by nearbycoder** [comment]
Definitely makes more sense now. Still thrown off by the _binding to backing_, but thats more because I have never heard it mentioned that way. Going ahead and assigning this to Kurt to be merged to master and deployed when he wants to push the article live.

**Pull request merged**